modules/cluster/src/main/java/io/fluo/cluster/runner/YarnAppRunner.java

```diff
@@ -290,8 +290,8 @@ public void kill(FluoConfiguration config) throws Exception {
   }
 
   /**
-   * Attempts to retrieves ResourceReport until maxWaitMs time is reached.
-   * Set maxWaitMs to -1 to retry forever.
+   * Attempts to retrieves ResourceReport until maxWaitMs time is reached. Set maxWaitMs to -1 to
```

Comment:
I committed these autoformatter changes in a4fbea8950f.  I wonder why GitHub is not complaining that your branch is stale.